doc/releases/luminous.rst

```diff
@@ -1,6 +1,103 @@
 v12.2.11 Luminous
 =================
 
+This is the twelfth bug fix release of the Luminous v12.2.x long term stable
+release series. We recommend that all users upgrade to this release.
+
+Notable Changes
+---------------
+
+* In 12.2.9 and earlier releases, keyring caps were not checked for validity,
+  so the caps string could be anything. As of 12.2.10, caps strings are
+  validated and providing a keyring with an invalid caps string to, e.g.,
+  `ceph auth add` will result in an error.
```

Comment:
I believe this release note was written with the assumption that the PR https://github.com/ceph/ceph/pull/24906 would be merged for the v12.2.10 release, but in reality it was merged for the v12.2.12 release so I think the release numbers should be changed accordingly.

@tchaikov Do you agree?

Reply:
@smithfarm thanks for pointing this out. yeah, we should s/12.2.9/12.2.11/ here.

Reply:
good catch, updating